src/components/EventModal/Collect/index.vue

```diff
@@ -37,16 +37,69 @@
           @click.once="onClickModelViewer"
         />
       </client-only>
-      <Label
-        class="text-medium-gray mt-[12px]"
-        preset="h6"
-        align="center"
-        :text="$t(
-          hasConnectedWallet
-            ? 'tx_modal_status_complete_text_collect'
-            : 'tx_modal_status_complete_text_collect_without_wallet'
-        )"
-      />
+      <Label class="text-medium-gray mt-[12px] flex-nowrap" preset="h6" align="center">
+        <i18n
+          :path="
+            hasConnectedWallet
+              ? 'tx_modal_status_complete_text_collect'
+              : 'tx_modal_status_complete_text_collect_without_wallet'"
+        >
+          <NuxtLink
+            class="font-[600] underline"
+            place="portfolio"
+            :to="localeLocation({
+              name: 'id___en',
```

Comment:
Why adding language suffix?
```suggestion
              name: 'id___en',
```